DOCS/man/options.rst

```diff
@@ -1148,6 +1148,27 @@ Watch Later
     Ignore path (i.e. use filename only) when using watch later feature.
     (Default: disabled)
 
+Watch History
+-------------
+
+``--save-watch-history``
+    Whether to save which files are played. These can be then selected with the
+    default ``g-h`` key binding.
+
+    .. warning::
+
+        This option may expose privacy-sensitive information and is thus
+        disabled by default.
+
+``--watch-history-path=<path>``
+    The path in which to store the watch history. Default:
+    ``~~state/watch_history.jsonl`` (see `PATHS`_).
+
+    This file contains one JSON array per line. The first field is the UNIX
+    timestamp when the file was opened, the second field is its normalized path,
+    and the third file is its title, or missing if no title was present. Any new
```

Comment:
title should be empty string if not available

Reply:
You can also show how it looks like `[timestamp, "file_path", "title"]`

Reply:
> But yeah we can just use keys if predictable column order is an issue.

I think we should try that, it will be easier to read and update this format.